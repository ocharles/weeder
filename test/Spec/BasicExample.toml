roots = [ "Spec.BasicExample.BasicExample.root" ]

type-class-roots = true
