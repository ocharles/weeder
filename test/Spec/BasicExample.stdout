test/Spec/BasicExample/BasicExample.hs:4: unrelated
